.github/workflows/functional-test.yaml

```diff
@@ -21,6 +21,8 @@ on:
     - cron: "30 0,4,8,12,16,20 * * 1-5"
     # Run every 12 hours on weekends.
     - cron: "30 0,12 * * 0,6"
+  # Allow manual dispatch
+  workflow_dispatch:
```

Comment:
Do we need to also accept parameters here? Like the branch name?

Where were you thinking we'd configure that?

See example of how we configure this in other cases here: https://github.com/radius-project/radius/pull/7294/files#diff-c79f364a9293abaaa8595776b74674e24bec6287834e63ab8aa7aec6a42f0dbcR101